releasenotes/notes/0.15/0.15.0-release-c7d85921f85f0b4d.yaml

```diff
@@ -0,0 +1,14 @@
+---
+prelude: >
```

Comment:
for multiple paragraph entry, you used the `|` format. Is that the same?

Reply:
ah, good catch thanks. Whitespace management in yaml is always a pain. Right now this is being treating as a single paragraph when rendered. I'll update it

Reply:
If you're curious the rules around this in yaml are kinda of ridiculous :) https://yaml.org/spec/1.2/spec.html#Block

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/4850/commits/8f56a03c244ea9a5fe2521b8b21359678f383a80